script.user.js

```diff
@@ -1617,8 +1617,8 @@ const Ruler = {
         xhr: true,
       },
       {
-        u: '||imagevenue.com/img.php',
-        q: '#thepic',
+        u: '||imagevenue.com',
```

Comment:
Anchor the tail like this: `u: '||imagevenue.com/'`

Reply:
Done. 👍